packages/card/README.md

```diff
@@ -0,0 +1,50 @@
+# Card
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/card.svg)
+
+## Example
+
+```Javascript
+import Card from '@leafygreen-ui/card';
+
+<Card
+  as='section'
+  className='card-styles'
+>
+  This is my card component
+</Card>
+```
+
+**Output HTML**
+
+```HTML
+ <section class="leafygreen-ui-1xf4bcx">This is a card component</section>
```

Comment:
Maybe `<article>` is a better semantic tag to use here. @hswolff Do you have a preference?

Reply:
It's defaulting to div, section was just an example of using the as prop.

Reply:
I don't have a strong preference.

I always feel weird using article as it's so semantically tied to news sites, which doesn't apply to most web apps.

Not sure which description on mdn applies best.